apps/mobile-wallet/src/components/layout/ScrollScreen.tsx

```diff
@@ -69,8 +68,6 @@ const ScrollScreen = ({
   const scrollEndHandler = useAutoScrollOnDragEnd(viewRef)
   const insets = useSafeAreaInsets()
 
-  useScrollToTopOnBlur(viewRef)
-
```

Comment:
I removed this to fix the annoying jump in the Settings screen happening every time we navigate to a sub-screen. Any objections?

Reply:
Sounds good to me.